Makefile

```diff
@@ -7,8 +7,8 @@ CHANGED_FILES_IN_BRANCH := $(shell git diff --name-only $(shell git merge-base o
 .PHONY : install_deps setup pre-commit pre-commit-in-branch pre-commit-all test help
 
 install_deps:  ## Install all dependencies.
-	pip install -r dev-requirements.txt
 	pip install -r requirements.txt
+	pip install -r dev-requirements.txt
```

Comment:
Should we create one command `pip install -r requirements.txt -r dev-requirements.txt` ?

Reply:
we can, I just had to change the order as I prefer to install core stuff lib first then the rest.